setup.py

```diff
@@ -44,7 +44,7 @@
     'mongolock>=1.3.4,<1.4',
     'PyYAML>=4.2b1,<5.0',
     'lxml>=3.8,<4.2',
-    'python-twitter==3.3',
+    'python-twitter==3.5',
```

Comment:
v3.3 has **App auth** but does not compatible with python 3, see [commit](https://github.com/bear/python-twitter/commit/3ee3c5a55b903e0bb62d3388c0ed594f45de5b35)
v3.5 has [fix typo](https://github.com/bear/python-twitter/commit/eed985f1cb5cd3dadfbd42732bb02c319a4f4653) that cause `'set' object does not support indexing`. This also because sometime twitter api return 0, not sure why and we could not reproduce it